### DISCUSSION TITLE
文章中引入代码默认每一行有行号

### DISCUSSION THREAD

**Discussion by flyly0755** [pylixm/django-mdeditor#161] [Q&A]
<img width="761" alt="image" src="https://user-images.githubusercontent.com/45039060/161763512-4fcce8bc-871e-4716-8e45-2379d3db181d.png">
请问，如何设置，可以不显示出行号🙏

**Comment by pylixm** [comment]
这个参数就是空时行号的。
![image](https://user-images.githubusercontent.com/15826317/182769678-67073cce-7019-4918-9efd-930cf6ac957b.png)
